root/static/scripts/guess-case/utils.js

```diff
@@ -219,11 +257,17 @@ exports.titleStringByMode = function (is, forceCaps) {
 
     if (doCaps) {
         var chars = os.split('');
-        chars[0] = chars[0].toUpperCase();
+        if(gc.mode.isTurkishMode())
+          chars[0] = TurkishUpperCase(chars[0]);
+        else
+          chars[0] = chars[0].toUpperCase();
 
         if (is.length > 2 && is.substring(0, 2) === 'mc') {
             // Make it McTitled
-            chars[2] = chars[2].toUpperCase();
+            if(gc.mode.isTurkishMode())
+              chars[2] = TurkishUpperCase(chars[2]);
+            else
+              chars[2] = chars[2].toUpperCase();
```

Comment:
While this works just fine, it has the drawback of cluttering the code up with `if`…`else` conditions.
IMHO, a more elegant way to do it would be to make a single call `gc.mode.toUpperCase(chars[2])` instead.
See related comment.